File: linaauddk/34691.py
import sys
input = sys.stdin.readline

info = {
    "animal": "Panthera tigris",
    "tree": "Pinus densiflora",
    "flower": "Forsythia koreana"
}
while True :
    question = input().strip()

    if question == "end":
        break
    if question in info :
        print(info[question])

File: yht0827/34691.java
// Main.java

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		while (true) {
			String s = br.readLine();
			if (s == null)
				break;
			if (s.equals("end"))
				break;

			if (s.equals("animal")) {
				System.out.println("Panthera tigris");
			} else if (s.equals("tree")) {
				System.out.println("Pinus densiflora");
			} else if (s.equals("flower")) {
				System.out.println("Forsythia koreana");
			}
		}
	}
}
